www/cgit/files/patch-config-make-uname.diff

```diff
@@ -0,0 +1,13 @@
+--- ./git/config.mak.uname.orig	2020-08-18 05:34:19.000000000 +0200
++++ ./git/config.mak.uname	2020-08-18 05:43:50.000000000 +0200
+@@ -133,8 +133,8 @@
+ 	HAVE_BSD_SYSCTL = YesPlease
+ 	FREAD_READS_DIRECTORIES = UnfortunatelyYes
+ 	HAVE_NS_GET_EXECUTABLE_PATH = YesPlease
+-	BASIC_CFLAGS += -I/usr/local/include
+-	BASIC_LDFLAGS += -L/usr/local/lib
++	BASIC_CFLAGS += -I/opt/local/include
++	BASIC_LDFLAGS += -L/opt/local/lib
```

Comment:
You can't assume that the user has installed MacPorts into /opt/local.

Reply:
MacPorts already adds the right `-I` and `-L` flags for whatever the user's MacPorts prefix is, so you can probably just remove the /usr/local lines here without replacing them with anything.

Reply:
yes, I was thinking about that after I made the diff. It would be enough just to drop those lines.